tests/test_black.py

```diff
@@ -310,6 +310,24 @@ def test_detect_pos_only_arguments(self) -> None:
         versions = black.detect_target_versions(root)
         self.assertIn(black.TargetVersion.PY38, versions)
 
+    def test_detect_debug_f_strings(self) -> None:
```

Comment:
Can you also include a couple of normal cases like the ones in here (not all of them, a couple should be enough)

https://github.com/python/cpython/blob/71c3d649b5a0324c6eb01f9ad025c1e102b82bba/Lib/test/test_fstring.py#L1262-L1291